pdbx/__init__.py

```diff
@@ -15,47 +18,44 @@
 from .writer import PdbxWriter
 from .errors import PdbxSyntaxError, PdbxError  # noqa: F401
 from .containers import DataCategory, DataContainer  # noqa: F401
-from ._version import get_versions
-
-__version__ = get_versions()["version"]
-del get_versions
+from .version import __version__  # noqa: F401
 
 
-def load(fp) -> list:
+def load(fobj) -> list:
```

Comment:
Can't see how `fobj` is a superior name to `fp`... but whatever :-)

Reply:
It just needs to be 3 letters to get past flake8/pylint warnings. The comments were that `fp` is a file object so `fobj` was an easy fix. I could have used `f_ptr` but thought that might look too much like `C/C++` :-)